### DISCUSSION TITLE
Possibility to rename/hide games files in the games library

### DISCUSSION THREAD

**Discussion by chewie-21** [OnionUI/Onion#1160] [Ideas]
Hello guys, love your incredible work on OnionOS !
Is there a possibility to add a feature to rename the games files in the games library interface ? and also an option menu to hide some games in the UI (a new menu entry when we click on X in the games library for example) ?
Thank you !

**Comment by Rahhide** [comment]
I think this good idea, in the GLO menu, maybe you change the title in  game list options
